README.md

```diff
@@ -14,20 +14,27 @@ documented in the `.msg` or `.srv` file associated with that interface.
 ROS messages and services used across many ROS packages in the project.
 
 ### Project-wide External Interfaces
-
+<!---
+Formatting reminder:
+1. Keep 'Type' column organized alphabetically
+2. 'mock' version of topic should be after its common counterpart
+--->
 | Topic                  | Type           | Publisher                | Subscriber(s)                               |
 | ---------------------- | -------------- | ------------------------ | ------------------------------------------- |
-| `ais_ships`            | AISShips       | AIS Receiver             | Local Pathfinding                           |
-| `batteries`            | Batteries      | CAN Transceiver          | Local Transceiver                           |
+| `ais_ships`            | AISShips       | CANTransceiverROSIntf    | Local Pathfinding, Local Transceiver        |
+| `mock_ais_ships`       | AISShips       | Mock AIS                 | CanSimIntf                                  |
+| `batteries`            | Batteries      | CANTransceiverROSIntf    | Local Transceiver                           |
+| `boat_cmd`             | BoatCmd        | Controller               | CANTransceiverROSIntf                       |
 | `desired_heading`      | DesiredHeading | Local Pathfinding        | Controller, Boat Simulator                  |
-| `data_sensors`         | GenericSensors | CAN Transceiver          | Local Transceiver                           |
-| `global_path`          | Path           | Local Transceiver        | Local Pathfinding                           |
-| `gps`                  | GPS            | CAN Transceiver          | Local Transceiver, Local Pathfinding        |
+| `data_sensors`         | GenericSensors | CANTransceiverROSIntf    | Local Transceiver                           |
+| `gps`                  | GPS            | CANTransceiverROSIntf    | Local Transceiver, Local Pathfinding        |
+| `mock_gps`             | GPS            | Boat Simulator           | CanSimIntf                                  |
 | `local_path_data`      | LPathData      | Local Pathfinding        | Local Transceiver                           |
-| `mock_gps`             | GPS            | Boat Simulator           | CAN Transceiver                             |
-| `filtered_wind_sensor` | WindSensor     | CAN Transceiver          | Local Transceiver, Local Pathfinding        |
-| `mock_wind_sensors`    | WindSensors    | Boat Simulator           | CAN Transceiver                             |
-| `wind_sensors`         | WindSensors    | CAN Transceiver          | Local Transceiver                           |
+| `global_path`          | Path           | Local/Remote Transceiver | Local Pathfinding                           |
+| `sim_cmd`              | SimCmd         | CanSimIntf               | Simulator                                   |
```

Comment:
What is `sim_cmd` for?

Reply:
Changes to make after our discussion:
- Both `boat_cmd` and `sim_cmd` should be the same type, `SailCmd`
- Can rename `boat_cmd` and `sim_cmd` to `sail_cmd` and `sim_sail_cmd`, respectively
- @evannawfal will update the contents of `SailCmd` in a another PR

Reply:
It's a placeholder along with `boat_cmd`, to be replaced in future with exact fields. `sim_cmd` is for the can simulation interface while `boat_cmd `is for the can transceiver

Reply:
Updated readme and the .msg

Reply:
@colinli02 There was a change in plans. Can you remove `sim_sail_cmd` from this PR. @evannawfal will be creating a PR with a new message. When we are ready to merge it, we will tag you in the PR so you know about the topic name and type.

Reply:
Ok, I can hold on this PR for now, just to check @DFriend01 do you mean remove just `sim_sail_cmd` from the readme or all of its assosciated type SailCmd (`sim_sail_cmd`, `sail_cmd` from readme and `SailCmd.msg`?)

Reply:
Upon looking at your current implementation, I do like that the types are the same. I think it will be advantageous to just merge this PR, but I want to add a change to one of the `.msg` files.

If we change the `uint` to a `float`, is that okay?

Reply:
This is what @evannawfal had:

```bash
# Angle to rotate the trim tab relative to the mailsail
# Unit: degrees, 0° is the neutral position, Increases CW
# Range: -40° <= angle <= 40°
float32 angle
```

@colinli02 if you could change the contents of `SailCmd.msg` to the above, then this PR is good to go from the perspective of the @UBCSailbot/controller team.

Reply:
Alright, so instead of removing `sim_sail_cmd` I have changed `SailCmd.msg` to that specified code:
```
# Angle to rotate the trim tab relative to the mailsail
# Unit: degrees, 0° is the neutral position, Increases CW
# Range: -40° <= angle <= 40°
float32 trim_tab_angle_degrees
```

Let me know if there's anything else @DFriend01, if it's all good I will resolve this conversation and merge it in

Reply:
LGTM